data-median.h

```diff
@@ -0,0 +1,92 @@
+#ifndef __DATA_MEDIAN__
+#define __DATA_MEDIAN__
+
+#define SIZE 9u
+
+int data[SIZE];
+uint8_t current_sample = 0;
+
+void count(double* a, double* inter, int* key, size_t size_a, size_t size_inter)
+{
+    for (size_t i = 0; i < size_a; ++i)
+    {
+        for (size_t j = 0; j < size_inter; ++j)
+        {
+            if (j != 0) {
+                if (a[i] < inter[j] && a[i] > inter[j-1]){
+                    key[j]++;
+                }
+            }
+            else {
+                if (a[i] < inter[j]){
+                    key[0]++;
+                }
+            }
+        }
+    }
+}
+
+void genVector(int* x, double* ri, double* inter, int* out, size_t size_ri, size_t size_inter)
+{
+    for (size_t i = 0; i < size_ri; ++i)
+    {
+        for (size_t j = 0; j < size_inter; ++j)
+        {
+            if (j != 0)
+            {
+                if (ri[i] < inter[j] && ri[i] > inter[j-1]){
+                    out[i] = x[j];
+                }
+            }
+            else{
+                if (ri[i] < inter[j]){
+                    out[i] = x[j];
+                }
+            }
+        }
+    }
+}
+size_t getSeed()
+{
+    return time(0);
+}
+void genRi(int k, double* ri)
+{
+    srand(time(0));
```

Comment:
Shouldn't getseed be used here?

Reply:
getSeed function will be removed. cause we do not use it. 
srand(time(0)) is used instead of getSeed func.
getSeed was used in first version of random numbers generator.